yt/utilities/lib/quad_tree.pyx

```diff
@@ -108,8 +121,10 @@ cdef class QuadTree:
                   int nvals, bounds, method = "integrate"):
         if method == "integrate":
             self.combine = QTN_add_value
-        elif method == "mip":
+        elif method in ["mip","max"]:
```

Comment:
I think this may cause a perf regression as it uses a Python idiom that doesn't translate directly to C. I suggest keeping `==` comparisons here

Reply:
Sounds good! Should be better now :)